protocol-units/bridge/contracts/src/AtomicBridgeCounterpartyMOVE.sol

```diff
@@ -57,8 +57,6 @@ contract AtomicBridgeCounterpartyMOVE is IAtomicBridgeCounterpartyMOVE, OwnableU
         uint256 amount
     ) external onlyOwner returns (bool) {
         if (amount == 0) revert ZeroAmount();
-        if (atomicBridgeInitiatorMOVE.poolBalance() < amount) revert InsufficientMOVEBalance();
```

Comment:
Without this check, how are we sure that there's enough MOVE available to cover the transfer?

Reply:
Because you can only bridge back enough MOVE that was bridged before and all bridged MOVE is contained in the eth side of the contracts.

Reply:
Yeah Primata is correct, you don't need poolBalance to do this check, the tx will revert if the contract doesn't have enough MOVE balance using the standard ERC20 checks

Reply:
Besides that revert, we have to work with the assumption that locked amount on Ethereum contract equals to circulating supply on Movement.

Reply:
On Movement Barcok yes. Let's set this correct amount on the new deployments.

Reply:
For now on Bardock that wouldn't be true due to the ability to use the Bardock faucet, but for mainnet I see how that reasoning makes sense.